tests/data/test_dataloader.py

```diff
@@ -40,6 +39,8 @@
     build_text_dataloader,
 )
 from llmfoundry.data.utils import get_tokens_per_batch_func
+# yapf: enable
+from llmfoundry.data_prep import convert_dataset_hf
 from llmfoundry.utils.builders import build_tokenizer
 from llmfoundry.utils.config_utils import to_dict_container
 # yapf: disable
```

Comment:
Why are you disabling yapf?

Reply:
https://github.com/mosaicml/llm-foundry/blob/502eb12ff40c69b8a7d693ace8120057afd34338/tests/data/test_dataloader.py#L45
Not sure, was always there

Reply:
I see, moving yapf back